core-web/src/main/java/org/silverpeas/core/webapi/sharing/TicketResource.java

```diff
@@ -120,7 +124,19 @@ private void validateUserAuthorizationOn(String componentId) {
   }
 
   private SharingTicketService getFileSharingService() {
-    return SharingServiceProvider.getSharingTicketService();
+    return getSharingTicketService();
   }
 
+  private PaginationPage fromPage(String page) {
+    PaginationPage paginationPage = null;
+    if (page != null && !page.isEmpty()) {
+      String[] pageAttributes = page.split(";");
+      int nth = Integer.valueOf(pageAttributes[0]);
+      int count = Integer.valueOf(pageAttributes[1]);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use "Integer.parseInt" for this string-to-int conversion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS2130)